src/App.js

```diff
@@ -28,6 +30,25 @@ class App extends Component {
 		this.setState({ currentUser: '', id: '' });
 	};
 
+	findMovieShowInfo = () => {
+		let movieID = window.location.pathname.split('/')
+		if(!this.state.foundMovie || this.state.foundMovie.id !== movieID){
+		fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieID[2]}`)
+			.then(response => {
+				if (response.ok) {
+				return response.json()
+			}})
```

Comment:
I wonder if it would be beneficial to turn this into an async/await function? Would that help solve what we were encountering yesterday with multiple fetch calls happening depending on where a user refreshed their page and throwing errors? Food for thought and research.